src/test/resources-noncompilable/com/puppycrawl/tools/checkstyle/checks/javadoc/javadocparamorder/InputJavadocParamOrderRecords.java

```diff
@@ -0,0 +1,73 @@
+//non-compiled with javac: Compilable with Java15
+package com.puppycrawl.tools.checkstyle.checks.javadoc.javadocparamorder;
+
+import java.util.List;
+import org.w3c.dom.Node;
+
+/**
+ * Config: default
+ * Some Javadoc.
+ *
+ * @since something
+ */
+class InputJavadocParamOrderRecords { // ok
```

Comment:
Please add some cases of violations in (constructors) and on records.

Reply:
done, please check

Reply:
Resolved.